engine/src/flutter/shell/platform/darwin/ios/framework/Source/FakeUIPressProxy.swift

```diff
@@ -0,0 +1,51 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import UIKit
+
+@testable import InternalFlutterSwift
+
+/// UIPressProxy subclass for use to create fake UIPress events in tests.
+@available(iOS 13.4, *)
+@objc
+public class FakeUIPressProxy: UIPressProxy {
+  private let dataPhase: UIPress.Phase
+  private let dataKey: UIKey  // Store the copied key
+  private let dataType: UIEvent.EventType
+  private let dataTimestamp: TimeInterval
+
+  @objc override public var phase: UIPress.Phase {
+    return dataPhase
+  }
+
+  @objc override public var key: UIKey? {
+    return dataKey
+  }
+
+  @objc override public var type: UIEvent.EventType {
+    return dataType
+  }
+
+  @objc override public var timestamp: TimeInterval {
+    return dataTimestamp
+  }
+
+  @objc public init(
+    phase: UIPress.Phase, key: UIKey, type: UIEvent.EventType, timestamp: TimeInterval
```

Comment:
nit: these should be separate lines per swift style guide

Reply:
This was the output of `swift-format`. Is there a swift-format flag to split these up?

Reply:
i don't think swift-format handles this. maybe it's extra requirement by google's style guide